renku/cli/_providers/__init__.py

```diff
@@ -31,20 +34,26 @@ class ProviderFactory:
     def from_uri(uri):
         """Get provider type based on uri."""
         is_doi_ = is_doi(uri)
-        if is_doi_ is False:
+        if is_doi_ is None:
             url = urlparse(uri)
             if bool(url.scheme and url.netloc and url.params == '') is False:
                 return None, 'Cannot parse URL.'
 
+        uri_lower = uri.lower()
+
         provider = None
-        if 'zenodo' in uri:
+        if 'zenodo' in uri_lower:
```

Comment:
```suggestion
        if 'zenodo' in uri.lower():
```

Reply:
Good catch! That was left over from an earlier version of the code where I reused `uri_lower` a couple of times. Thanks